docs/guide/integrations.asciidoc

```diff
@@ -11,6 +11,467 @@ You can find integration options and information on this page.
+{ "index": {}}
+{"birth_date":"1954-09-16T00:00:00Z","emp_no":"10096","first_name":"Jayson","gender":"M","hire_date":"1990-01-14T00:00:00Z","languages":"4","last_name":"Mandell","salary":"43889","height":"1.94","still_hired":"false","avg_worked_seconds":"204381503","job_positions":["Architect","Reporting Analyst"],"is_rehired":["false","false","false"]}
+{ "index": {}}
+{"birth_date":"1952-02-27T00:00:00Z","emp_no":"10097","first_name":"Remzi","gender":"M","hire_date":"1990-09-15T00:00:00Z","languages":"3","last_name":"Waschkowski","salary":"71165","height":"1.53","still_hired":"false","avg_worked_seconds":"206258084","job_positions":["Reporting Analyst","Tech Lead"],"is_rehired":["true","false"],"salary_change":"-1.12"}
+{ "index": {}}
+{"birth_date":"1961-09-23T00:00:00Z","emp_no":"10098","first_name":"Sreekrishna","gender":"F","hire_date":"1985-05-13T00:00:00Z","languages":"4","last_name":"Servieres","salary":"44817","height":"2.00","still_hired":"false","avg_worked_seconds":"272392146","job_positions":["Architect","Internship","Senior Team Lead"],"is_rehired":"false","salary_change":["-2.83","8.31","4.38"]}
+{ "index": {}}
+{"birth_date":"1956-05-25T00:00:00Z","emp_no":"10099","first_name":"Valter","gender":"F","hire_date":"1988-10-18T00:00:00Z","languages":"2","last_name":"Sullins","salary":"73578","height":"1.81","still_hired":"true","avg_worked_seconds":"377713748","is_rehired":["true","true"],"salary_change":["10.71","14.26","-8.78","-3.98"]}
+{ "index": {}}
+{"birth_date":"1953-04-21T00:00:00Z","emp_no":"10100","first_name":"Hironobu","gender":"F","hire_date":"1987-09-21T00:00:00Z","languages":"4","last_name":"Haraldson","salary":"68431","height":"1.77","still_hired":"true","avg_worked_seconds":"223910853","job_positions":"Purchase Manager","is_rehired":["false","true","true","false"],"salary_change":["13.97","-7.49"]}
+--------------------------------------------------
+// TEST[skip:TBD]
+====
+
+[discrete]
+[[convert-dataset-pandas-dataframe]]
+==== Convert the dataset
+
+Use the ES|QL CSV import to convert the `employees` dataset to a Pandas
+dataframe object. 
+
+[source,python]
+------------------------------------
+from io import StringIO
+
+from elasticsearch import Elasticsearch
+import pandas as pd
+
+client = Elasticsearch(
+    "https://[host].elastic-cloud.com",
+    api_key="...",
+)
+
+response = client.esql.query(
+    query="FROM employees | LIMIT 500",
+    format="csv",
+)
+df = pd.read_csv(StringIO(response.body))
+print(df)
+------------------------------------
+
+However, the dataset only contains 100 records a LIMIT command is usesd to avoid
+getting ES|QL warnings about potentially missing records. This prints the
```

Comment:
```suggestion
Even though the dataset contains only 100 records, a LIMIT of 500 is specified to prevent
ES|QL warnings about potentially missing records. This prints the
```

Reply:
(or at least I *think* this is what this sentence is saying)